transaction/core/src/tx.rs

```diff
@@ -438,6 +430,18 @@ impl core::convert::From<[u8; 32]> for TxOutConfirmationNumber {
     }
 }
 
+impl core::convert::From<RistrettoPublic> for TxOutConfirmationNumber {
```

Comment:
I believ this should be `From<&RistrettoPublic>`, there's no need for it to consume it.

Reply:
Done.